### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by CaptnCodr**
Update + formatting.

FAKE isn't quite working on our setup here, I'm analyzing this.

### CODE REVIEW

**Comment by CaptnCodr** [comment]
Release note version now needs to have four digits (0.0.0.0) instead of three (0.0.0), else a wrong version will be taken.

This PR closes #227, #228, #229, #230.

**Comment by sergey-tihon** [comment]
@CaptnCodr what kind of incompatibility with .NET 7 did you found?

File: paket.lock [hunk 1]
```diff
@@ -75,18 +75,18 @@ NUGET
...
+      System.Runtime.CompilerServices.Unsafe (>= 4.5.3) - restriction: || (&& (== net6.0) (>= monotouch)) (&& (== net6.0) (>= net461)) (&& (== net6.0) (< netcoreapp2.0)) (&& (== net6.0) (< netcoreapp2.1)) (&& (== net6.0) (< netstandard1.1)) (&& (== net6.0) (< netstandard2.0)) (&& (== net6.0) (>= uap10.1)) (&& (== net6.0) (>= xamarinios)) (&& (== net6.0) (>= xamarinmac)) (&& (== net6.0) (>= xamarintvos)) (&& (== net6.0) (>= xamarinwatchos)) (== netstandard2.0)
+    System.Numerics.Vectors (4.5) - restriction: || (&& (== net6.0) (>= net462)) (== netstandard2.0)
     System.Reflection (4.3)
       Microsoft.NETCore.Platforms (>= 1.1)
       Microsoft.NETCore.Targets (>= 1.1)
       System.IO (>= 4.3)
       System.Reflection.Primitives (>= 4.3)
       System.Runtime (>= 4.3)
-    System.Reflection.Metadata (7.0) - restriction: || (== net6.0) (== net7.0) (&& (== netstandard2.0) (>= netcoreapp3.1))
+    System.Reflection.Metadata (7.0.1) - restriction: || (== net6.0) (&& (== netstandard2.0) (>= netcoreapp3.1))
```
Comment (sergey-tihon) [inline paket.lock:89]: Are you sure that it is OK to build for .net 6 and requite `System.*` assemblies with v7+ version?

**Comment by CaptnCodr** [comment]
@sergey-tihon FAKE 6 cannot run build.fsx using dotnet 7+ as a build runner. 
Then we need step back to dotnet 6 or get rid of FAKE.

File: paket.lock (hunk 1 above)
Comment (CaptnCodr) [reply paket.lock:89]: Seems correct. I deleted all caches and reinstalled all references.

**Review by sergey-tihon — APPROVED** [review]

**Pull request merged**